app/uk/gov/hmrc/securemessage/controllers/model/cdsf/read/ApiLetter.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage.controllers.model.cdsf.read
+
+import org.joda.time.DateTime
+import play.api.libs.json.{ Format, Json }
+import play.api.libs.json.JodaReads.jodaDateReads
+import play.api.libs.json.JodaWrites.jodaDateWrites
+import reactivemongo.bson.BSONObjectID
+import uk.gov.hmrc.securemessage.models.core.Letter
+
+final case class ApiLetter(
+  subject: String,
+  content: String,
+  firstReaderInformation: FirstReaderInformation,
+  readTime: Option[DateTime] = None
+)
+
+final case class FirstReaderInformation(name: Option[String], read: DateTime)
+
+object ApiLetter {
+  def fromCore(letter: Letter): ApiLetter =
+    ApiLetter(
+      letter.subject,
+      letter.content,
+      FirstReaderInformation(None, DateTime.now())
+    )
+
+  private val dateFormatString = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
```

Comment:
This is already used in a number of other places should we consider defining in one place?